lib/rules/no-identical-title.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var utils = require("../utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function isFirstArgLiteral(node) {
+    return node.arguments && node.arguments[0] && node.arguments[0].type === "Literal";
+}
+
+
+function handlTestCaseTitles(context, titles, node, title) {
+    if (utils.isTest(node.callee)) {
+        if (titles.indexOf(title) !== -1) {
+            context.report({
+                node: node.arguments[0],
+                message: "Test title is used multiple times in the same test module."
+            });
+        }
+        titles.push(title);
+    }
+}
+
+function handlTestSuiteTitles(context, titles, node, title) {
+    if (!utils.isModule(node.callee)) {
+        return;
+    }
+    if (titles.indexOf(title) !== -1) {
+        context.report({
+            node: node.arguments[0],
+            message: "Test suite title is used multiple times."
```

Comment:
Nitpick, but I'd love to see this say "Module name" instead of "Test suite title".